aerospike-core/src/cluster/partition.rs

```diff
@@ -36,15 +36,15 @@ impl<'a> Partition<'a> {
     }
 
     pub fn new_by_key(key: &'a Key) -> Self {
-        let mut rdr = Cursor::new(&key.digest[0..4]);
+        let mut rdr = Cursor::new(&key.digest[0..2]);
```

Comment:
This shouldn't actually make any difference. I was just following what the C client does when I noticed a problem that appeared to be related to sharding (but wasn't)... in theory it should be slightly more efficient.